nnsvs/model.py

```diff
@@ -25,7 +24,6 @@ def __init__(self, dim, dilation=1):
             WNConv1d(dim, dim, kernel_size=1),
         )
         self.shortcut = WNConv1d(dim, dim, kernel_size=1)
-
```

Comment:
minor: could you keep the new line here?

Reply:
I added new line there in the latest commit.